### ISSUE TITLE
Upload the file please

### ISSUE THREAD

**Issue by danielroehrig** [rominronin/katana-plate#1]
I just ordered my second katana60 from CandyKeys (because it's my favorite keyboard so far) but the plate is sold out. Can we have your scad file to order a plate myself?

**Comment by rominronin** [comment]
Hi Daniel,
I just read the email - I have some pretty heavy real life things going on atm.

I don’t have the original plate file, with the katana symbol in the middle. But I can put one together without for you - that’s all I have time for at the moment I’m afraid.

Baris

**Comment by danielroehrig** [comment]
Don't want to bother you. I just saw the repo and thought, you might have simply forgotten to push after a local commit. If you find the time to do upload a plate with or without a symbol that before the pandemic ends and I need to go back to the office, that would be awesome. But of course real life is more important. Take care!

**Comment by rominronin** [comment]
Of course.

Well there’s always the diy option - keyboard layout editor combined with swill’s online plate builder (Google them) - this is what I normally use as a base and it works perfectly fine, provided you know exactly which bottom row you want.

**Comment by danielroehrig** [comment]
Yeah, I 3D printed and hand soldered a couple of keyboards with that combination. I could do that. What about the screw positions? Is this a default setting Swill for the screw positions for a 60% layout?

**Comment by rominronin** [comment]
Yes exactly - 60% boards are just about the  only standardised size in the community, which is why I wanted to support it - gives you a tonne of case options.

The Katana60 layout has one switch that overlaps with a screw post, my recommendation is to trim the back of the screw so it is flush with the back of the PCB

**Comment by danielroehrig** [comment]
Update: I laser cutted (sic) a plate out of brass and it looks cool. I had to replace the katana symbol with three dots because I'm lazy. It works, but the space bar stabs might be a bit tight. I had to order two plates, because there was no cost difference between one or two plates. Do you want to have my second plate?

**Comment by rominronin** [comment]
Sure, thanks, why not!
I can pay for shipping - my PayPal address is ***@***.***

**Comment by danielroehrig** [comment]
That didn't work. github censors your email.  So, I can't give you mine either. Awesome. Also, the email in your profile doesn't work. I mean, you could go to my website (https://www.a-broad-view.com/impressum/), look for my email and send your address there...

**Comment by rominronin** [comment]
Let’s try that again!

***@***.*** is my PayPal address.

Thanks

**Comment by danielroehrig** [comment]
Github doesn't like email addresses

**Comment by danielroehrig** [comment]
The thing is, that your email me at baristosun dot com bounces. I can't even answer to your emails directly. Try this: send your post address to mail at danielroehrig dot de

**Comment by danielroehrig** [comment]
Now, both emails that I have of you bounce with "too many hops (in reply to end of DATA command)". The second one forwards to the first one and that never worked for me. Anyway, plate is in the mail. Have fun and take care!

**Comment by danielroehrig** [comment]
Did you get it? I mean, it was just a big envelope with no tracing, so I wouldn't be too surprised if it were still somewhere between Germany and Austria.

**Comment by rominronin** [comment]
I did, thank you. It came quite quickly, but I never got round to responding. Are you sure you don’t want anything for it?

I got your feedback on a katana update - my free time has dropped significantly and keyboards weren’t even my number one hobby, so the time I have for new developments is extremely limited.

I have a couple of follow up PCBS in the works that are years in the making - that’s how slow the progress is right now.

I fully take your feedback about the glowing backlight though - that would be a really cool addition.

**Comment by danielroehrig** [comment]
Cool! I was worried it got lost in the mail. I now own two katanas and I think I will buy up one of the two remaining PCBs from CandyKeys because no matter how hard I try, that left hand layout of the katana just makes so much more sense and is so much more comfortable than any "normal" layout and I can't move back. So, I hope that at some point there will be another run. Hot-swapable and with an indicator LED and I'll be very happy.
